src/Pages/PayPage/Pay.js

```diff
@@ -0,0 +1,40 @@
+import React, { Component } from 'react';
+import './Pay.scss';
+import PayContainer from './PayContainer';
+
+const typeList = ['delivery', 'order'];
+
+export default class Pay extends Component {
+  render() {
+    return (
+      <div className="pay">
+        <div className="header">
+          <i className="fas fa-chevron-left"></i>
```

Comment:
```suggestion
          <i className="fas fa-chevron-left" />
```
셀프 클로징 태그 적용 가능한 곳은 가급적 적용해주세요

Reply:
넵 감사합니다.!